Apps/CommonData/src/Validations/DependantAgeValidator.cs

```diff
@@ -0,0 +1,42 @@
+// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+
+namespace HealthGateway.Common.Data.Validations
+{
+    using System;
+    using FluentValidation;
+
+    /// <summary>
+    /// Validates a dependent's age against a maximum value using date of birth.
+    /// </summary>
+    public class DependantAgeValidator : AbstractValidator<DateTime>
```

Comment:
Is there or can there be a unit test associated with this validator?

Reply:
added unit tests